karmada-CLI-installtion-and-deploy-example/intro.md

```diff
@@ -0,0 +1,43 @@
+### Background：
+
+1. The kubeconfig files for the Karmada cluster, member1 cluster, and member2 cluster are located in the $HOME/.kube directory. These files are named config, config-member1, and config-member2 respectively.
```

Comment:
We don't usually say `Karmada cluster`, if you mean the cluster where installing Karmada, I'd prefer to use `host cluster`.

Reply:
Thank you for pointing out